### ISSUE TITLE
Responsive Design

### ISSUE THREAD

**Issue by ank1traj** [abhpd-team/OEP-Frontend#41]
The website is not fully responsive with different screen sizes.

**Comment by abhpd** [comment]
Yes, that's why we're bootstrapping. Check out this branch [Bootstrapping](https://github.com/abhpd-team/OEP-Frontend/tree/Bootstrapping)
